cypress/integration/happy-path-with-accessibility.js

```diff
@@ -146,9 +146,27 @@ describe('Get help to retrain smoke test', function() {
     checkAccessibility();
   });
 
-  it('should allow me to find local schemes and offers', function() {
```

Comment:
could we keep the local schemes step and add a new cv advice step

Reply:
oops. fixed now.